### PR TITLE
Handling ddp training tests

### PR DESCRIPTION
**Pull request by SiddhantSadangi**
# Description

* Added ddp code to scheduled tests
* Ignoring ddp code in PR tests

__Related to:__ `DDP training examples`

---

## ❔ This change

- [ ] adds a new feature
- [ ] fixes breaking code
- [X] is cosmetic (refactoring/reformatting)

---

## ✔️ Pre-merge checklist

Mark which items have been completed:

- [ ] Refactored code ([sourcery](https://sourcery.ai/))
- [ ] Tested code locally
- [X] Added code to GitHub tests ([notebooks](workflows/test-notebooks.yml), [scripts](workflows/test-scripts.yml))
- [ ] Updated GitHub [README](../README.md)

### 🪝 Run the following to both [pre-commit](https://pre-commit.com/) hooks and JupyterLab (turn on `format on save`)

- [ ] Stripped outputs from notebooks - ([nbstripout](https://pypi.org/project/nbstripout/))
- [ ] Sorted imports - ([isort](https://pycqa.github.io/isort/))
- [ ] Formatted code - ([back](https://github.com/psf/black))

---

## 🧪 Test Configuration

- OS:
- Python version:
- Neptune version:
- Affected libraries with version:

### CODE REVIEW

File: .github/workflows/test-scripts.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ jobs:
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
         scripts: # Add in alphabetical order
           - how-to-guides/data-versioning/scripts
+          - how-to-guides/ddp-training/scripts
```
Comment (Blaizzy) [inline .github/workflows/test-scripts.yml:17]: This is just setup for when GH gets the GPU instances, right?

Comment (SiddhantSadangi) [reply .github/workflows/test-scripts.yml:17]: yeah, just a placeholder till then

Comment (Blaizzy) [reply .github/workflows/test-scripts.yml:17]: Great!

**Review by Blaizzy — APPROVED** [review]
LGTM

**Pull request merged**